SCF/SCF-Session9-GraphicsProblem/Screen.java

```diff
@@ -0,0 +1,100 @@
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.TreeMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+public class Screen {
```

Comment:
You can have a single method to sort will take parameter sortBy, override methods accordingly.